src/lightning_lite/strategies/utils.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
This file is in the strategies directory, but wouldn't a precision dir be more appropriate?

Reply:
yes, why not